test/lib/usd/utils/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(TARGET_NAME DiffCore)
+set(TARGET_NAME Usd_Utils_GTests)
```

Comment:
Renaming the project name to Usd_Utils_GTests since it is more generic and we will later have more tests added here.

Also, the directory structure under test/usd needs a bit more attention. We have a mix of python and c++ tests here. It makes sense to have these moved to it's own sub-directories ( e.g python, gtest ). Something for another day.